flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
This should be removed. We need to ensure that the appdata is valid and correctly installed.